### PR TITLE
 [release] Increment version to 2.15.0

### PR DESCRIPTION
**Pull request by cdietrich**
[release] Increment version to 2.15.0

### CODE REVIEW

File: releng/releng-target/xtext-core.target.target [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.8"?>
-<target name="org.eclipse.xtext.core.target" sequenceNumber="0">
+<target name="org.eclipse.xtext.helios.target" sequenceNumber="0">
```
Comment (kthoms) [inline releng/releng-target/xtext-core.target.target]: this change is produced by the generateP2Build task and should be reverted

Comment (cdietrich) [reply releng/releng-target/xtext-core.target.target]: will d you that or should i?
we fix the generate p2 build task then

Comment (kthoms) [reply releng/releng-target/xtext-core.target.target]: OK, we'll put this on our list

Comment (kthoms) [reply releng/releng-target/xtext-core.target.target]: Tracked in https://github.com/eclipse/xtext/issues/1220

**Pull request merged**